core/libraries/batch/JobHandlers/RegistrationsReport.php

```diff
@@ -257,10 +270,11 @@ public function get_csv_data_for($event_id, $offset, $limit, $question_labels, $
                 $reg_csv_array = array();
                 if (! $event_id) {
                     // get the event's name and Id
-                    $reg_csv_array[ __('Event', 'event_espresso') ] = sprintf(
+                    $reg_csv_array[ (string) __('Event', 'event_espresso') ] = sprintf(
+                        /* translators: 1: event name, 2: event ID */
                         __('%1$s (%2$s)', 'event_espresso'),
```

Comment:
what's the point of even localizing this?  If all it contains is some brackets and dynamic values?

Reply:
I think translators  might want to reorder them, or use something besides brackets. (Eg RTL languages might want to put them in the other order?) I'm not positive on exact cases but the justifications for it are pretty weak. But I'm inclined to leave it, especially considering folks might be using that translation to filter the value too.